shell/platform/common/cpp/client_wrapper/include/flutter/event_channel.h

```diff
@@ -0,0 +1,116 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Please split event channel support out into a separate precursor PR to keep the scope smaller.

Reply:
Ok, I will submit event_channel.h in another PR.